Kimseho_assignment/app.js

```diff
@@ -6,8 +6,17 @@ const cors = require("cors");
 const morgan = require("morgan");
 
 const { DataSource } = require("typeorm");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
 
-const database = new DataSource({
+
+const saltRounds = 10;
+const header = {
+  typ: "JWT",
+  alg: "HS256",
+};
+const payLoad = { foo: "bar" };
+const myDataSource = new DataSource({
```

Comment:
세호님 ! 이전에 과제에서 리뷰 드린 것과 같이 myDataSource의 경우 어떤 데이터 값을 불러오는지 의미 파악이 어려울 수 있어 아래 suggetion이나 혹은 어떠한 데이터를 불러오는지에 대해 고민해 보시고 수정해 주시면 좋을 듯 합니다!
```suggestion
const appDataSource = new DataSource({
```